lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -72,14 +72,20 @@ export function extractPackageFile(
   const endpoint = GlobalConfig.get('endpoint');
   try {
     // TODO: use schema (#9610)
-    const doc = parseSingleYaml<GitlabPipeline>(replaceReferenceTags(content));
-    const includes = getAllIncludeProjects(doc);
-    for (const includeObj of includes) {
-      const dep = extractDepFromIncludeFile(includeObj);
-      if (platform === 'gitlab' && endpoint) {
-        dep.registryUrls = [endpoint.replace(regEx(/\/api\/v4\/?/), '')];
+    const docs = parseYaml<GitlabPipeline>(replaceReferenceTags(content), {
+      uniqueKeys: false,
+    });
+    for (const doc of docs) {
+      if (is.object(doc)) {
+        const includes = getAllIncludeProjects(doc);
+        for (const includeObj of includes) {
+          const dep = extractDepFromIncludeFile(includeObj);
+          if (platform === 'gitlab' && endpoint) {
+            dep.registryUrls = [endpoint.replace(regEx(/\/api\/v4\/?/), '')];
```

Comment:
please extract this to outer loop, so we don't need to do the replace again and again